server/src/main/proto/com/handcontrol/server/request.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+package com.handcontrol.server;
+
+service HandleRequest{
+  rpc login(LoginRequest) returns (LoginResult);
+  rpc ProRequest(ClientRequets) returns (ClientReply);
+}
+message LoginRequest {
```

Comment:
@Pavlouha one mobile has one proId ? should we pass it in login?

Reply:
ProId is the id of prostheses part. I want to specific it.

Reply:
Yea , but I am thinking about how better to get info about protheseses and mobiles connected. Are they many to one, or one to one